stroeer/navigation/v1/navigation_service.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+package stroeer.navigation.v1;
+
+import "stroeer/core/v1/shared.proto";
+
+option go_package = "github.com/stroeer/go-tapir/navigation/v1;navigation";
+
+service NavigationService {
+  // Retrieve navigation tree structure
+  rpc GetNavigation(GetNavigationRequest) returns (GetNavigationResponse) {}
```

Comment:
`rpc GetNavigation(google.protobuf.Empty) returns (GetNavigationResponse) {}`

and then add a message later if required?